HarfangLab/harfanglab/ingest/parser.yml

```
@@ -43,6 +52,9 @@ stages:
           host.domain: "{{json_event.message.agent.domainname}}"
           host.os.version: "{{json_event.message.agent.osversion}}"
           host.os.full: "{{json_event.message.agent.osproducttype}}"
+          harfanglab.groups: "{{json_event.message.groups}}"
+          harfanglab.log_type: "{{json_event.message.log_type}}"
```

After consideration, I wonder if `event.dataset` is not more suitable?
```suggestion
          event.dataset: "{{json_event.message.log_type}}"
```